spec/controllers/high_voltage/pages_controller_spec.rb

```diff
@@ -3,16 +3,13 @@
 RSpec.describe HighVoltage::PagesController do
   render_views
 
-  # Test that each static page is generated correctly
-  Dir.glob(Rails.root.join('app', 'views', 'pages', '*.erb')).each do |file|
-    page_name = file.split('/').last.split('.').first
-
+  %w[ cookies terms_and_conditions unsubscribe ].each do |page_name|
     describe "when rendering #{page_name}" do
       before do
         get :show, id: page_name
       end
 
-      it { should respond_with(:success) }
+      specify { expect(response.status).to eq(200) }
```

Comment:
Agreed. We're already using these matchers elsewhere (in `healthcheck_controller_spec.rb`).